### DISCUSSION TITLE
Hobby-Teams

### DISCUSSION THREAD

**Discussion by Jak2k** [vercel/community#1600] [Idea]
Hobby-Teams for easier grouping and bulk-upgrading of projects.

**Comment by amyegan** [comment]
Nice suggestion! Can you tell me more about what you want to see, or what you want it to accomplish for you? I don't want to make any assumptions about your intent when I bring it up with the team 😄

**Comment by Jak2k** [comment]
I think it would be very practical to group project into free Hobby-Teams, because then I could upgrade all projects in the team at once, when I need to do, without affecting unrelated projects.
